### ISSUE TITLE
Possibility to support BCM43602?

### ISSUE THREAD

**Issue by vmarkovtsev** [seemoo-lab/nexmon#56]
Hi,

I wonder what do you think about the possibility to apply the patch for BCM43602 (installed in MacBooks).

**Comment by matthiasseemoo** [comment]
Yes, if you want to spent some time to reverse engineer the function locations, I can give you an introduction.

**Comment by jekkos** [comment]
OK great, any documentation around on how to do this? Which blob should be analyzed?

**Comment by xionglingfeng** [comment]
Any updates on this? @matthiasseemoo Do you mind putting the introduction somewhere online so everyone could access? I am interested in BCM43602 too.

**Comment by JuniorJPDJ** [comment]
I'm interested in introduction to help self-patching, but more generic, not only for BCM43602.
